batch/batch/worker/worker.py

```diff
@@ -261,8 +261,7 @@ def __init__(self, job, name, spec):
             tag = 'latest'
 
         if repository in HAIL_GENETICS_IMAGES:
-            repository_name_without_prefix = repository[len(HAIL_GENETICS):]
-            repository = f'gcr.io/{PROJECT}/{repository_name_without_prefix}'
+            repository = f'gcr.io/{PROJECT}/{repository}'
```

Comment:
The batch pool executor tests will catch if this is wrong -- correct?

Reply:
Yeah, the images would be absent and that would cause an error. I've already uploaded the images to GCR using the first version of this code. I just ran it again now to ensure the new thing is right too.